src/main/java/com/github/dockerjava/api/DockerClient.java

```diff
@@ -251,6 +256,48 @@
 
     DisconnectFromNetworkCmd disconnectFromNetworkCmd();
 
+    /**
+     * Enables swarm mode for the docker engine and creates a new swarm cluster
+     *
+     * @since 1.24
+     * @param swarmSpec the specification for the swarm
+     * @return the command
+     */
+    InitializeSwarmCmd initializeSwarmCmd(SwarmSpec swarmSpec);
```

Comment:
See https://github.com/KostyaSha/yet-another-docker-plugin/blob/1931326a58c43093f3aff5e5c993ada2eb46f8c9/yet-another-docker-its/src/test/java/com/github/kostyasha/it/tests/NginxRegistryTest.java#L62-L115

About order as i understand all tests are not ordered by it's nature. 
If swarm mode doesn't affect standard docker-engine mode, then it's fine.
It also maybe possible to do something in scopes of Suite afair or testng has something specific...